scripts/util/doc-builder.js

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env node
+
+var path = require('path');
+var fs = require('fs');
+var yuidoc = require('./yuidoc');
+var NodeGit = require('nodegit');
+var util = require('util');
+var fsExtra = require('fs-extra');
+
+function DocBuilder(options) {
```

Comment:
Hey @lemonmade for some reason I can't view the original commit. But I'm guessing the above code was something like:

```
function DocBuilder(options) {
  var objectThatBuildsDocs = {};

  return objectThatBuildsDocs;
}
```

Should there be a preference for classes at Shopify over functions that return objects? I've always liked doing the above since you can instantiate either by calling as a function or using a the `new` operator.